bridge-token-factory/src/lib.rs

```diff
@@ -31,8 +27,6 @@ const BRIDGE_TOKEN_NEW: Gas = 10_000_000_000_000;
 /// Initial balance for the BridgeToken contract to cover storage and related.
 const BRIDGE_TOKEN_INIT_BALANCE: Balance = 30_000_000_000_000_000_000_000_000; // 3e25yN, 30N
```

Comment:
Should this be reduced 10 times?

Reply:
yep, March 8 on MainNet.

Reply:
According to my experiment calculations it should be increased 🤔 

cost per byte on testnet is: 100000000000000000000 (is the cost already decreased in testnet?)
contract size in bytes is: 3769047 (Notice that it is growing in this PR from 481014 to 3769047 almost 10x)

```
BRIDGE_TOKEN_INIT_BALANCE = 30000000000000000000000000
MINIMUM_EXPECTED          = 376904700000000000000000000
```